src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.querySelector('#largeImg');
+const listImg = document.querySelector('#thumbs');
+
+listImg.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target;
+
+  if (!target) {
+    return;
+  }
+
+  if (target.tagName === 'IMG') {
+    mainImg.src = target.parentElement.href;
```

Comment:
Accessing `target.parentElement.href` assumes that the parent element is always an anchor (`<a>`) tag. Ensure that this assumption is valid in your HTML structure to avoid potential errors.